proposals/field-keyword.md

```diff
@@ -687,6 +681,10 @@ partial class C
 
 **Recommendation**: Permit an initializer on either part of a partial property when the implementation part uses `field`. Report an error if both parts have an initializer.
 
+#### Answer
+
+Allow `field` in implementing declaration and allow initializer in defining declaration (which compels implementation to be field-backed).
```

Comment:
@cston Should I move this back to the Open Questions section? I'm not sure whether we confirmed that initializers would be allowed on either part (not just the defining part) or that it would be an error if both parts have an initializer (even when set to the same constant, etc).

Reply:
Think this section should move to having a more full example that helps demonstrate why we chose to allow this behavior.

Reply:
> I'm not sure whether we confirmed that initializers would be allowed on either part

From [LDM-2024-08-26](https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-08-26.md#conclusion-1), an initializer is allowed on either part, but not both.

Reply:
Updating using the wording from the official notes!